<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>A</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>B</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>score</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>status</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip">
      <property name="text">
       <string>http://127.0.0.1:3000/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="set_ip">
      <property name="text">
       <string>set ip</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutools">
    <property name="title">
     <string>tools</string>
    </property>
    <addaction name="calc"/>
    <addaction name="lev1"/>
    <addaction name="lev2"/>
    <addaction name="lev3"/>
    <addaction name="lev4"/>
    <addaction name="reload"/>
   </widget>
   <addaction name="menutools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="calc">
   <property name="text">
    <string>calculate</string>
   </property>
  </action>
  <action name="lev1">
   <property name="text">
    <string>deactivate leverage 1</string>
   </property>
  </action>
  <action name="lev2">
   <property name="text">
    <string>activate leverage 2</string>
   </property>
  </action>
  <action name="reload">
   <property name="text">
    <string>reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="lev3">
   <property name="text">
    <string>activate lev3</string>
   </property>
  </action>
  <action name="lev4">
   <property name="text">
    <string>lev4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
